### ISSUE TITLE
`stack` auto-analysis getting stuck

### ISSUE THREAD

**Issue by mumbel** [NationalSecurityAgency/ghidra#5400]
**Describe the bug**
When running auto-analysis, the `stack` option gets stuck on a function and doesn't progress.  Can cancel and start over and its the same index each time.  I have tried with all sub-options turned off, no difference.  It got stuck for a pretty long time, same index, I killed it then thought to do jstack.

PowerPC:BE:32:QUICC (1.5)

**Attachments**
immediately after noticing bad index [stack1.txt](https://github.com/NationalSecurityAgency/ghidra/files/11607844/stack1.txt)
a few minutes in [stack2.txt](https://github.com/NationalSecurityAgency/ghidra/files/11607846/stack2.txt)
a bit longer wait [stack3.txt](https://github.com/NationalSecurityAgency/ghidra/files/11607847/stack3.txt)

**Environment (please complete the following information):**
 - OS: 22.04
 - Java Version: 17.0.7
 - Ghidra Version: 10.3
 - Ghidra Origin: official

I can do eclipse and load on master if that would be helpful, just let me know.

**Comment by emteere** [comment]
Is the function where it gets stuck calling a function that has been changed to inline?

**Comment by mumbel** [comment]
I have made one inline function that just returns the previous frames LR.  I'll check that tonight if that was in the function.  This one prints the function along with updating the index, is there any easy way to get the corresponding function for the mips based on index, or will I need to run that in eclipse

**Comment by emteere** [comment]
To test, I would just try selecting and analyzing the one function that calls the inlined function or just analyze the program again.

If you mean the monitor progress index, there really isn't a correspondence between the function and the index, especially with potentially multiple threads running.

It is possibly there is some waffling setting the flow override for the mips, similar to a bug on ARM fixed in 10.3.
If there is any inline of functions, my best guess is that it is from the inline.

**Comment by mumbel** [comment]
I unchecked inline for the function and seems to fix both this and the constant analyzer.

**Comment by emteere** [comment]
Great thanks for checking it out!  Glad it wasn't something new.
We'll fix in a patch release soon.

**Comment by mumbel** [comment]
```
         80 81 00 00     lwz        r4,0x0(r1)=>Stack[0x0]
         80 64 00 04     lwz        r3,Stack[0x4](r4)
         38 63 ff fc     subi       r3,r3,0x4
         4e 80 00 20     blr
```
for reference.  master w/ inline finishes `stack` analysis

**Comment by mumbel** [comment]
10.3.1_patch looks good w/ inline

**Comment by ryanmkurtz** [comment]
Fixed by bf8d314ddb5822a1b1663dca7f5c7ca3052c2d9e
